packages/editor/src/components/index.js

```diff
@@ -23,7 +23,7 @@ export {
 	RichTextShortcut,
 	RichTextToolbarButton,
 	RichTextInserterItem,
-	RichTextInputEvent,
```

Comment:
`RichTextInputEvent` was included in 5.1 RC1 (#13833, #13908). Do we need to cherry-pick this into 5.1 final? Or do a full deprecation?

cc @noisysocks

Reply:
It's meant to be included before the release. I don't mind not doing it, but atm this is more meant to be an internal thing and I'd rather not have it seen documented as stable API. Only thought about it after merge, sorry :/

Reply:
Let's include this in 5.1 and avoid deprecations.